packages/errors/src/messages.ts

```diff
@@ -137,6 +141,12 @@ export const SolanaErrorMessages: Readonly<{
         'here: https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts',
     [SOLANA_ERROR__SUBTLE_CRYPTO_SIGN_FUNCTION_MISSING]: 'No signing implementation could be found',
     [SOLANA_ERROR__SUBTLE_CRYPTO_VERIFY_FUNCTION_MISSING]: 'No key export implementation could be found',
+    [SOLANA_ERROR__TRANSACTION_INVALID_NONCE_TRANSACTION_FIRST_INSTRUCTION_NOT_ADVANCE_NONCE]:
+        'Transaction first instruction is not advance nonce account instruction.',
```

Comment:
I know I almost certainly put these periods here, but we should probably either _always_ end error messages with periods, or _never_ end them with periods.